lib/libmbedtls/core/tee_lmd_provider.c

```diff
@@ -1505,57 +1516,349 @@ TEE_Result crypto_acipher_dh_shared_secret(struct dh_keypair *private_key,
 #endif /* CFG_CRYPTO_DH */
 
 #if defined(CFG_CRYPTO_ECC)
-TEE_Result
-crypto_acipher_alloc_ecc_public_key(struct ecc_public_key *s __unused,
-				    size_t key_size_bits __unused)
+
+TEE_Result crypto_acipher_alloc_ecc_keypair(struct ecc_keypair *s,
+					    size_t key_size_bits)
 {
-	return TEE_ERROR_NOT_IMPLEMENTED;
+	memset(s, 0, sizeof(*s));
+	s->d = crypto_bignum_allocate(key_size_bits);
+	if (!(s->d))
```

Comment:
Please remove the redundant inner () here and at the other similar places below.